Google.PowerShell.IntegrationTests/BigQuery/BigQuery.GcbqDataset.Tests.ps1

```diff
@@ -0,0 +1,85 @@
+﻿. $PSScriptRoot\..\BigQuery\GcbqCmdlets.ps1
+$project, $zone, $oldActiveConfig, $configName = Set-GCloudConfig
+
+Describe "New-GcbqDataset" {
+
+	It "should take a name, description, and time to make a dataset" {
+		$data = New-GcbqDataset "test_data_id1" -Name "Testdata" -Description "Some interesting data!" -Timeout 86400000
+		$data | Should Not BeNullOrEmpty
+		$data.DatasetReference.DatasetId | Should Be "test_data_id1"
+		$data.FriendlyName | Should Be "Testdata"
+		$data.Description | Should Be "Some interesting data!"
+		$data.DefaultTableExpirationMs | Should Be 86400000
+    }
+
+	It "should accept a dataset object from pipeline" {
+		$data = New-Object -TypeName Google.Apis.Bigquery.v2.Data.Dataset
+		$data.DatasetReference = New-Object -TypeName Google.Apis.Bigquery.v2.Data.DatasetReference
+        $data.DatasetReference.DatasetId = "test_data_id2"
```

Comment:
Spacing on this line is not consistent with the others. Same for a few other lines in this file.